src/map/CMakeLists.txt

```diff
@@ -137,9 +137,9 @@ set(SOURCES
 )
 
 if(UNIX)
-    set(resource "")
+    set(resource conquest_zmq.cpp conquest_zmq.h conquest_data.cpp conquest_data.h)
 else()
-    set(resource "${CMAKE_SOURCE_DIR}/res/mapserver.rc")
+    set(resource "${CMAKE_SOURCE_DIR}/res/mapserver.rc" conquest_zmq.cpp conquest_zmq.h conquest_data.cpp conquest_data.h)
```

Comment:
These don't go here, these go in the SOURCES section above - this is just for the rc  icon

Reply:
My IDE put them there.. will move em out.